vars/runBenchmarkTestScript.groovy

```diff
@@ -38,6 +39,12 @@
  * @param args.userTag <optional> - Additional metadata tags to be added to benchmark run metrics, e.g., run-type:adhoc,arch:x64
  * @param args.configName <optional> - Name of the config file that needs to be downloaded from S3 bucket, default is config.yml.
  * @param args.telemetryParams <optional> - Allows to set parameters for telemetry devices such as node-stat etc., e.g. {"node-stats-include-indices": "true"}
+ * @param args.baseline <required> - The baseline TestExecution ID used to compare the contender TestExecution.
+ * @param args.contender <required> - The TestExecution ID for the contender being compared to the baseline.
+ * @param args.results_format <optional> - Defines the output format for the command line results, either markdown or csv. Default is markdown.
+ * @param args.results_numbers_align <optional> - Defines the column number alignment for when the compare command outputs results. Default is right.
+ * @param args.results_file <optional> - When provided a file path, writes the compare results to the file indicated in the path.
+ * @param args.show_in_results <optional> - Determines whether or not to include the comparison in the results file.
```

Comment:
Add an `||` condition on line 62 to check if `args.command == 'execute-test` as we don't need that `config` for compare runs.